src/ss_script_handler.py

```diff
@@ -291,7 +292,10 @@ def iterate_once(self):
             elif "PAUSE" in elements[0] and len(elements) >= 2:
                 self.logger.debug("PAUSE")
                 try:
+                    self.logger.debug('now pausing')
                     time.sleep(int(elements[1]))
+                    #rospy.sleep(int(elements[1]))
+                    self.logger.debug('after pausing') #cmhuang: never got here when using one single launch file...
```

Comment:
Why was this committed? What is the issue here with pausing? If there is an actual issue, please open an issue with more details so we can fix it!